datahub-frontend/app/controllers/TrackingController.java

```diff
@@ -4,66 +4,46 @@
 import client.AuthServiceClient;
 import com.fasterxml.jackson.databind.JsonNode;
 import com.typesafe.config.Config;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.List;
-import java.util.Optional;
-import java.util.Properties;
 import javax.annotation.Nonnull;
 import javax.inject.Inject;
-import org.apache.kafka.clients.CommonClientConfigs;
-import org.apache.kafka.clients.producer.KafkaProducer;
-import org.apache.kafka.clients.producer.ProducerConfig;
+import javax.inject.Singleton;
+
+
 import org.apache.kafka.clients.producer.ProducerRecord;
-import org.apache.kafka.common.config.SaslConfigs;
-import org.apache.kafka.common.config.SslConfigs;
-import org.apache.kafka.common.security.auth.SecurityProtocol;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import play.mvc.Controller;
 import play.mvc.Http;
 import play.mvc.Result;
 import play.mvc.Security;
-import utils.ConfigUtil;
+import client.KafkaTrackingProducer;
 
-import static auth.AuthUtils.*;
+import static auth.AuthUtils.ACTOR;
 
 
 // TODO: Migrate this to metadata-service.
```

Comment:
Is this comment still applicable?

Reply:
Out of scope for this PR, however forwarding directly to kafka reduces this overhead on GMS. So its arguable whether it should be done.